tests/test_nodes.py

```diff
@@ -42,7 +42,9 @@ def test_deleting_node(driver: WebDriver):
     ).click()
 
     # need to remove the comment once the bug on machinist page
-    # assert len(driver.find_elements(By.XPATH, r'.//*[contains(@class,"State__StateRoot")]')) == 1 and
+
+    # assert len(driver.find_elements(By.XPATH, r'.//*[contains(@class,"State__StateRoot")]')) == 1
```

Comment:
This is commented because of the extra node bug?

Reply:
yes when the bug is fixed we can uncomment the line and it should work

Reply:
If so, can you just add a note like this:

```python
     # TODO: below will be enabled when bug (FLAP-XXX) is fixed
     # assert len(driver.find_elements(By.XPATH, r'.//*[contains(@class,"State__StateRoot")]')) == 1 
```

Where `FLAP-XXX` is the issue for the extra node bug, I forgot which issue, its on jira